### DISCUSSION TITLE
虚心请教MPV的几个设置问题

### DISCUSSION THREAD

**Discussion by 906018080** [hooke007/mpv_PlayKit#177] [Q&A 问答] [answered]
1.无边框最大化貌似就是全屏 有没什么命令让无边框最大化时候显示任务栏

2.进度条上的章节标记,如何设置不显示

3.都是100音量的情况下  mpv音量比potplayer小好多

**Comment by hooke007** [comment, marked as answer]
1 无边框最大化不是全屏，无边框的窗口状态在windows系统下属于灯下黑状态无法遵循正常窗口的运行逻辑，所以我也不使用

2 osc 的脚本设置里如果没有对应选项就是不能

3 potplayer 默认不使用wasapi，默认启用内建的音频滤镜，默认启用回放增益？，所以实际音量不等

**Comment by 906018080** [comment]
好的,老大都说不行我也就不折腾了
